README.md

```diff
@@ -17,9 +24,40 @@ We record our decisions in the `docs/adr` folder. See the [first ADR](./docs/adr
 adr new Branching strategy
 ```
 
-## Name
+## Running Luminary
 
-lu·​mi·​nary - ˈlü-mə-ˌner-ē
+For Visual Studio Code users, `./.vscode/launch.json` includes debug configurations for the API, CMS and reference App.
 
-1. a person of prominence or brilliant achievement
-2. a body that gives light
+### API
+
+Run the API commands from the `./api` directory. See the `.env.example` file in the `./api` directory on creating an `.env` file.
+
+Before running Luminary against a clean CouchDB database it is recommended to seed the database with the default document set. This document set is also used for unit tests, and should help you to get a functional setup to start with.
+
+`npm run seed`
+
+Starting the api:
+
+`npm run start`
+
+Running unit tests:
+
+`npm run test` (once-off testing) or `npm run test:watch` (continous testing)
+
+### CMS
+
+Run the CMS commands from the `./cms` directory. See the `.env.example` file in the `./cms` directory on creating an `.env` file.
+
+Luminary includes a web-server for running the Content Management System web-app.
+
+Starting the webserver:
+
+`npm run dev`
+
+Running CMS unit tests:
+
+`npm run test`
```

Comment:
We have specific setup/running instructions in the readme's of the folders, perhaps add your new note on seeding there? The rest of this should already be in there.